pkg/controller/metrics/exporters/lvm_exporter/lvm_exporter.go

```diff
@@ -133,6 +222,7 @@ func validateCliArg(arg1 string) string {
 	}
 	return arg1
 }
+
 // main function for lvm exporter
 // lvm exporter is a independent process which user can start if required
 func main() {
```

Comment:
I don't know why there is a main method here.

Reply:
LVM Exporter is an independent process which user starts on demand. User can either use lvm exporter or periodic post API . This is why we have main() here.